solutions/usaco-920.mdx

```diff
@@ -9,6 +9,61 @@ author: Sofia Yang
 
 <LanguageSection>
 
+<CPPSection>
+	
+```cpp
+
+#include <iostream>
+#include <fstream>
+#include <vector>
+using namespace std;
+ 
+int N, M, answer;
+int L[100001];
+vector<int> S_nbrs[100001], D_nbrs[100001];
+bool impossible;
+ 
+void visit(int x, int v)
+{
```

Comment:
tabs instead of spaces

Reply:
This is not a stylistic choice. USACO Guide requires you to use tabs, not spaces.

Reply:
woah there buddy don't be so harsh
but yeah this is the *one* thing mods enforce lol